orchestrator-salt/src/main/resources/salt-common/salt/filecollector/scripts/preflight-check.sh

```diff
@@ -0,0 +1,338 @@
+#!/bin/sh
+
+EXPECTED_MIN_VERSION="0.4.13"
+# ~300MB
+MAX_FILE_SIZE=314572800
```

Comment:
can be MAX_FILE_SIZE_BM

Reply:
actually that's in bytes, so will keep as is

Reply:
then it can be MAX_FILE_BYTES so that there is no confusion about units